dom/dom_c.php

```diff
@@ -574,12 +574,12 @@ class DOMDocument extends DOMNode  {
     public $documentElement;
 
     /**
-     * @var string
+     * @var string|null
      * @since 5.0
      * The location of the document or NULL if undefined.
      * @link https://php.net/manual/class.domdocument.php#domdocument.props.documenturi
      */
-    public $documentURI ;
+    public $documentURI;
```

Comment:
```suggestion
    public $documentURI ;
```
This formatting change is unrelated to the fix

Reply:
I took the opportunity to fix this (in addition to the 2 formatting changes below) because they were on the same line, and make the code more consistent with the rest of the stubs. If someone from JetBrains wants to keep formatting changes in another PR, I'll happily accept your changes, otherwise I think it's doing its bit to improve the stub quality.

Reply:
Ok, let's leave it as it is, thx for your contribution. I suggest to add nullability to the following properties: DomNode.parentNode (with corresponding documentation update), DomNode.firstChild, DomNode.lastChild, DomNode.previousSibling, DomNode.nextSibling and the same properties for the DOMElement

Reply:
Very good point! I updated these properties in `DOMNode` + `DOMElement`, and also for [`DOMNode::$ownerDocument`](https://www.php.net/manual/en/class.domnode.php#domnode.props.ownerdocument). @AlexMovsesov do you work for JetBrains?

Reply:
Thx, yes, I am working in JetBrains